gems/aws-sdk-s3/spec/object/download_file_spec.rb

```diff
@@ -221,6 +221,58 @@ module S3
           end.to raise_error(Aws::Errors::ChecksumError)
         end
 
+        it 'does not download object when ETAG does not match during multipart get by ranges' do
+          allow(client).to receive(:head_object).with({
```

Comment:
I guess this is fine seeing as other tests in this file are doing it.. but the tests here should be refactored (transfer manager makes these kind of obsolete anyway).